marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/transforms/mlcp-flow-transform.sjs

```diff
@@ -74,12 +71,20 @@ function transform(content, context = {}) {
 
     const contentArray = buildContentArray(context);
 
-    if (runStepsInMemory(theFlow, options)) {
-      // Let errors propagate to MLCP
-      options.throwStepError = true;
-      flowApi.runFlowOnContent(flowName, contentArray, jobId, options);
+    // Have to tokenize on ";" since the transform param value already tokenizes on ","
+    const stepNumbers = params.steps ? params.steps.split(";") : null;
```

Comment:
Is this a standard practice for MLCP or do we should probably get this documented?

Reply:
It's DHF-specific - MLCP only allows for a single transform_param, so DHF is tokenizing it on "," to find multiple parameters in the value of transform_param. Since steps can now have multiple values, we need a second delimiter. This will be added to the DHF docs for 5.5.